finality-aleph/src/nodes.rs

```diff
@@ -57,7 +57,7 @@ where
     C: crate::ClientForAleph<Block, BE> + Send + Sync + 'static,
     C::Api: AlephSessionApi<Block> + AuraApi<Block, AuraId>,
     BE: Backend<Block> + 'static,
-    TP: TransactionPool<Block = Block> + 'static,
+    TP: TransactionPool<Block = Block, Hash = primitives::Hash> + 'static,
```

Comment:
Nit: you probably want `BlockHash` here, although I think it refers to the same thing underneath? Or maybe I'm very wrong about this, but then I would like to know. :P

Reply:
Hash here is the transaction hash, so I wanted to stress that. Anyway, `BlockHash = primitives::Hash = H256`, so this is only an aesthetic preference.

Reply:
If you want to stress the distinction then there should be a separate constant in `primitives` called `TransactionHash` – I think that's actually a nice idea, even though I believe there are some uses within the codebase that would strictly speaking be wrong because of that (no need to look for these in this PR, we will eventually find them or they don't matter enough).

Reply:
Done